app/views/settings/index.html.erb

```diff
@@ -16,6 +16,11 @@
     </li>
   <% end %>
 </ul>
+
+<% if @settings.count == 0 %>
```

Comment:
Isn't there something like `empty?`. I'd also use `-%>` to prevent it from rendering an empty line (and the same for the `end` tag):
```suggestion
<% if @settings.empty? -%>
```

Reply:
> <% if @settings.empty? -%>

Thanks @ekohl , that works :)